packages/react-components/react-tags-preview/src/contexts/useTagAvatarContextValues.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { AvatarSize, AvatarShape } from '@fluentui/react-avatar';
+
+export type UseTagAvatarContextValuesOptions = {
+  avatarSize: AvatarSize;
+  avatarShape: AvatarShape;
+};
+
+export type TagAvatarContextValues = {
+  avatar: {
+    size?: AvatarSize;
+    shape?: AvatarShape;
+  };
+};
+
+export function useTagAvatarContextValues_unstable(state: UseTagAvatarContextValuesOptions): TagAvatarContextValues {
```

Comment:
You still need to export this from `packages/react-components/react-tags-preview/src/index.ts` otherwsie no one will be able to re-compose this component properly

Reply:
lol I wasn't exporting any context 🤦‍♀️ thanks for finding this